src/styles/utils/placeholders.scss

```diff
@@ -0,0 +1,3 @@
+%basic-hover-animation {
+  transition: 300ms linear;
```

Comment:
`transition` is a shorthand property, use `transition-duration` if you only style the duration. And specify what properties you want to be affected by this rule.